### DISCUSSION TITLE
Hello I can't request the GitHub Student Pack

### DISCUSSION THREAD

**Discussion by AidenOfficial** [community/community#48986] [GitHub Education]
I'm a Chinese Student and my school didn't offer some files that the machine scan can identify. But I have the certification from CSSD (Center for Student Services and Development, Ministry of Education, P.R. China). I'd like to know if I can confirm my student status through human customer support, please!

**Comment by whitewatercn** [comment]
可以参考我的这个帖子再逝一逝 https://xlog.whitewater.wang/Github_Student_Developer_Pack
You can refer to this post to try again https://xlog.whitewater.wang/Github_Student_Developer_Pack
